tron/core/actionrun.py

```diff
@@ -800,14 +801,18 @@ def recover(self):
     def do_recover(self, delay):
         recovery_command = f"{self.action_runner.exec_path}/recover_batch.py {self.action_runner.status_path}/{self.id}/status"
 
+        # Put the "recovery" output at the same directory level as the original action_run's output
+        recovery_output_path = self.output_path.clone()
+        recovery_output_path.base = os.path.dirname(recovery_output_path.base)
+
         # Might not need a separate action run
         # Using for the separate name
         recovery_run = SSHActionRun(
-            job_run_id=self.job_run_id,
-            name=f"recovery-{self.id}",
+            job_run_id=f"{self.job_run_id}-recovery",
```

Comment:
hm, I don't know whether there are any side effects to this? is it used anywhere to link the recovery action with the original job run

Reply:
Not as far as I can tell.  I tested by:

1) starting a long-running job on `master`
2) killing tron
3) switching to this branch
4) restarting tron

and it recovered the job just fine.

Reply:
Why change the job_run_id from `self.job_run_id` to `self.job_run_id-recovery` anyway? It doesn't change the output path.

Reply:
I don't know, I'm reverting this.